src/adapters/desyn/execution/src/sdk/lib.ts

```diff
@@ -0,0 +1,29 @@
+import { JsonRpcProvider } from 'ethers';
+import { UserTVLData } from "./types";
+import axios from "axios"
+
+export const getTimestampAtBlock = async (blockNumber: number) => {
+  const provider = new JsonRpcProvider("https://rpc.zklink.io");
+  const block = await provider.getBlock(blockNumber);
+  return Number(block?.timestamp);
+};
+  
+export async function getUserTVLData(blockNumber: number):Promise<UserTVLData[]>  {
+
+  const response = (await axios.get("https://api.desyn.io/zklink/points_gateway/points/get_zklink_balance_list?blockNumber=" + blockNumber));
+  const data = response.data.list;
```

Comment:
Unable to request successfully, 404 error

Reply:
The api is reconditioned as GET

Reply:
I encountered a 500 error. Could you ensure the stability of the service? It's crucial for retrieving points accurately.
do you have the resources to develop a subgraph as a data source?

Reply:
I will keep the service stable, for the time being, we can only provide data through api form.